modules/cluster-logging-deploy-eo-cli.adoc

```diff
@@ -96,6 +96,7 @@ kind: Subscription
 metadata:
   generateName: "elasticsearch-"
   namespace: "openshift-operators-redhat" <1>
+  name: "elasticsearch-operator"
```

Comment:
@mburke5678 I think we don't need to add `name: "elasticsearch-operator"` since we already have `generateName: "elasticsearch-"`.

Reply:
@QiaolingTang It seemed to create the Subscription with the name I specified, when I specify both `name` and  `generateName`:

```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  generateName: "elasticsearch-"
  namespace: "openshift-operators-redhat"
spec:
  channel: "4.3"
  installPlanApproval: "Automatic"
  source: "redhat-operators"
  sourceNamespace: "openshift-marketplace"
  name: "elasticsearch-operator"
```

```
$ oc create -f eo-sub.yaml
subscription.operators.coreos.com/elasticsearch-jd68t created
$ oc get subscription.operators.coreos.com --all-namespaces
NAMESPACE                    NAME                  PACKAGE                  SOURCE             CHANNEL
openshift-operators-redhat   elasticsearch-jd68t   elasticsearch-operator   redhat-operators   4.3
```
But I will leave the docs as they are, without `name` and close this PR.

Reply:
But if you want to add `name: "elasticsearch-operator"`, please remove `generateName: "elasticsearch-"`.

Reply:
> @QiaolingTang It seemed to create the Subscription with the name I specified, when I specify both `name` and `generateName`:
> 
> ```
> apiVersion: operators.coreos.com/v1alpha1
> kind: Subscription
> metadata:
>   generateName: "elasticsearch-"
>   namespace: "openshift-operators-redhat"
>   name: "elasticsearch-operator"
> spec:
>   channel: "4.3"
>   installPlanApproval: "Automatic"
>   source: "redhat-operators"
>   sourceNamespace: "openshift-marketplace"
>   name: "elasticsearch-operator"
> ```
> 
> ```
If you put `name: "elasticsearch-operator"` on the top of `generateName: "elasticsearch-"`, then creating the sub, the name of sub should be `elasticsearch-operator`, so only the first name is valid, we don't need to keep two names.

Reply:
@QiaolingTang It seems to pick up the `name` value even when `generateName` is present and first in the list. 
I removed `generateName` to match what we have for the Cluster Logging Operator install. 
Thanks for all the help and the quick reviews!